pkgs/applications/networking/instant-messengers/cordless/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, buildGoModule, fetchurl }:
+
+buildGoModule rec {
+  name = "cordless-${version}";
+  version = "2019-06-13";
+
+  src = fetchurl {
```

Comment:
Please use ``fetchFromGitHub``.

Reply:
Sure, but may I ask why that's better?

Reply:
`fetchFromGitHub` is better to use when you don't rely on a release tarball because nix can do some sort of caching that it can't with `fetchurl`.

Reply:
Ah I see, so it caches parts that are the same instead of the whole tarball, which would be useless.